File: java/easy/Fizz_Buzz.java
package easy;
import java.util.*;

public class Fizz_Buzz {
	

	class Solution {
	    public List<String> fizzBuzz(int n) {
	        List<String> answer = new ArrayList<>();

	        for(int i = 1; i <= n; i++) {

	            if(i % 3 == 0 && i % 5 == 0) {
	                answer.add("FizzBuzz");
	                continue;
	            }

	            if(i % 3 == 0) {
	                answer.add("Fizz");
	                continue;
	            }

	            if(i % 5 == 0) {
	                answer.add("Buzz");
	                continue;
	            }

	            answer.add(i + "");
	        }


	        return answer;
	    }
	}
}

File: javascript/easy/Fizz_Buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
const fizzBuzz = function (n) {
  let resultArray = [];
  for (let i = 1; i <= n; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
      resultArray.push("FizzBuzz");
    } else if (i % 3 == 0) {
      resultArray.push("Fizz");
    } else if (i % 5 == 0) {
      resultArray.push("Buzz");
    } else {
      resultArray.push("" + i);
    }
  }
  return resultArray;
};
